internal/kms/kms.go

```diff
@@ -216,6 +221,344 @@ func (k *Kms) CreateKeys(ctx context.Context, scopeId string, opt ...Option) err
 	return nil
 }
 
+// ListKeys lists all keys in the scope.
+// Options are ignored.
+func (k *Kms) ListKeys(ctx context.Context, scopeId string, _ ...Option) ([]wrappingKms.Key, error) {
+	const op = "kms.(Kms).ListKeys"
+	keys, err := k.underlying.ListKeys(ctx, scopeId)
+	if err != nil {
+		if errors.Is(err, dbw.ErrRecordNotFound) {
+			return nil, errors.E(ctx, errors.WithCode(errors.RecordNotFound), errors.WithOp(op))
+		}
+		return nil, errors.Wrap(ctx, err, op)
```

Comment:
would seem pretty simple to test this error path...

Reply:
Good spot, added a test